enterprise/core-edge/src/main/java/org/neo4j/commandline/dbms/UnbindFromClusterCommand.java

```diff
@@ -0,0 +1,193 @@
+        }
+
+        @Override
+        public Optional<String> arguments()
+        {
+            return Optional.of( "[--database=<name>] " );
+        }
+
+        @Override
+        public String description()
+        {
+            return "Removes cluster state data from the specified database making it suitable for use non-Core-Edge " +
+                    "databases";
+        }
+
+        @Override
+        public AdminCommand create( Path homeDir, Path configDir, OutsideWorld outsideWorld )
+        {
+            return new UnbindFromClusterCommand( homeDir, configDir );
+        }
+    }
+
+    private Path homeDir;
+    private Path configDir;
+
+    UnbindFromClusterCommand( Path homeDir, Path configDir )
+    {
+        this.homeDir = homeDir;
+        this.configDir = configDir;
+        this.storeLockChecker = new StoreLockChecker();
+    }
+
+    @Override
+    public void execute( String[] args ) throws IncorrectUsage, CommandFailed
+    {
+        Args parsedArgs = Args.parse( args );
+
+        try
+        {
+            Config config = loadNeo4jConfig( homeDir, configDir, databaseNameFrom( parsedArgs ) );
+            Path pathToSpecificDatabase = config.get( DatabaseManagementSystemSettings.database_path ).toPath();
+
+            Validators.CONTAINS_EXISTING_DATABASE.validate( pathToSpecificDatabase.toFile() );
+            confirmClusterStateDirectoryExists( Paths.get( pathToSpecificDatabase.toString(), "cluster-state" ) );
+
+            try ( Closeable ignored = storeLockChecker.withLock( pathToSpecificDatabase ) )
+            {
+                deleteClusterStateIn( clusterStateFrom( pathToSpecificDatabase ) );
+            }
+            catch ( IOException e )
+            {
+                wrapIOException( e );
+            }
+
+        }
+        catch ( StoreLockException e )
+        {
+            throw new CommandFailed(
+                    "Database is currently locked. Please check that no other Neo4j instance using it.", e );
+        }
+        catch ( IllegalArgumentException e )
+        {
+            throw new IncorrectUsage( e.getMessage() );
+        }
+        catch ( UnbindFailureException | CannotWriteException e )
+        {
+            throw new CommandFailed( "Unbind failed: " + e.getMessage(), e );
+        }
+    }
+
+    private void wrapIOException( IOException e ) throws CommandFailed
+    {
+        throw new CommandFailed(
+                format( "Unable to unbind database: %s: %s", e.getClass().getSimpleName(), e.getMessage() ), e );
+    }
+
+    private Path clusterStateFrom( Path target )
+    {
+        return Paths.get( target.toString(), CLUSTER_STATE_DIRECTORY_NAME );
+    }
+
+    private void confirmClusterStateDirectoryExists( Path clusterStateDirectory ) throws UnbindFailureException
+    {
+        if ( !exists( clusterStateDirectory ) )
+        {
+            throw new UnbindFailureException( "Database %s is not bound to any cluster", clusterStateDirectory );
```

Comment:
Does this surface as an error? I would expect this command to be idempotent -- and therefore exit 0 in this case, possible (but not definitely) with a notification that nothing was done.